core/backend/src/ECDb.ts

```diff
@@ -57,6 +57,22 @@ export class ECDb implements Disposable {
     this._nativeDb.dispose();
     this._nativeDb = undefined;
   }
+  /**
+   * Attach a ECDb file to this connection and load and register its schemas.
+   * @param fileName ECDb file name
+   * @param alias identifier for the attached file. This identifer is used a tablespace executing ECSQL queries.
```

Comment:
I don't know what "a tablespace" is supposed to mean. Maybe you meant "as tablespace", but I don't recognize the term "tablespace". I know what this parameter does, but I don't know the proper way to describe it.

Reply:
remvoe tablespace here as well.